packages/docusaurus/src/commands/swizzle/tables.ts

```diff
@@ -74,11 +74,11 @@ function actionsTable(): string {
       logger.code('--wrap'),
       `
 Creates a wrapper around the original theme component.
-Allows rendering other components before/after the original theme component.
+Allows rendering of other components before/after the original theme component.
```

Comment:
```suggestion
Allows rendering other components before/after the original theme component.
```

This is actually correct 😄